packages/app/obojobo-express/server/migrations/20221214210826-modify-users-add-last-login-column.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+var dbm
+var type
+var seed
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+	dbm = options.dbmigrate
+	type = dbm.dataType
+	seed = seedLink
+}
+
+exports.up = function(db) {
+	return db.addColumn('users', 'last_login', {
+		type: 'timestamp WITH TIME ZONE',
+		notNull: true,
+		defaultValue: new String('now()')
+	})
+}
+
+exports.down = function(db) {
+	return null
```

Comment:
The `down` of a migration is the opposite of the `up`. In this case, the `down` of this migration should remove the `last_login` column.